roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Default values for docker role
+
+docker_version: 1.9.1
```

Comment:
ah, ok. `(1)` pretty much means that we can stick to 1.9.1 for now. thanks for clarifying @vvb